lib/spack/spack/cmd/clean.py

```diff
@@ -59,6 +60,9 @@ def setup_parser(subparser):
     subparser.add_argument(
         '-p', '--python-cache', action='store_true',
         help="remove .pyc, .pyo files and __pycache__ folders")
+    subparser.add_argument(
+        '-l', '--locks', action='store_true',
+        help="remove lock files (typically unnecessary)")
     subparser.add_argument(
         '-a', '--all', action=AllClean, help="equivalent to -sdmp", nargs=0
     )
```

Comment:
I think you need to add `args.locks` to the section below:

> # If nothing was set, activate the default

Otherwise, it will also clear the stage when you only run `--locks`.

Reply:
Please also add a warning, that this should _only_ be run when you are sure that no other process from this spack is active